src/Symfony/Component/HttpFoundation/File/MimeType/ExtensionGuesserInterface.php

```diff
@@ -27,5 +27,5 @@ interface ExtensionGuesserInterface
      *
      * @return string The guessed extension or NULL, if none could be guessed
      */
-    public function guess($mimeType);
+    public function guess(string $mimeType);
```

Comment:
all classes in the MimeType subnamespace are deprecated: they should be removed instead